_drafts/2015-05-26-welcome-to-jekyll.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "On Seminars"
+date:   2015-05-26 12:03:10
+categories: jekyll update
```

Comment:
Ah gotcha. I don't remember adding that - it might have been some automatic setting? I'll make sure to leave them out until needed for differentiation.